### DISCUSSION TITLE
How to have a Custom ReturnPath for email sent using  Postal?

### DISCUSSION THREAD

**Discussion by XenyFord** [postalserver/postal#1561] [Help with using Postal]
Hello there,

Can any of you please help me on how to make my postal settings to change where I can have the email's return path as "no-reply@mydomain.com" instead of xhstk@psrp.mydomain.com ? Or any other option available to have all the bouced emails forwarded to our bouce handling address i.e. No-reply@mydomain.com from postal?

Because, I have eveything setup within the Acymailing script to auto remove or unsubscribe/supression of users from my lists, and I don't want Postal to handle the bounce list. Or guide me if there is any other option to achieve it..

Thanks in advance for your kind help and support.!

Xeny

**Comment by XenyFord** [comment]
> Hello there,
> 
> Can any of you please help me on how to make my postal settings to change where I can have the email's return path as "[no-reply@mydomain.com](mailto:no-reply@mydomain.com)" instead of [xhstk@psrp.mydomain.com](mailto:xhstk@psrp.mydomain.com) ? Or any other option available to have all the bouced emails forwarded to our bouce handling address i.e. [No-reply@mydomain.com](mailto:No-reply@mydomain.com) from postal?
> 
> Because, I have eveything setup within the Acymailing script to auto remove or unsubscribe/supression of users from my lists, and I don't want Postal to handle the bounce list. Or guide me if there is any other option to achieve it..
> 
> Thanks in advance for your kind help and support.!
> 
> Xeny

Can any one help me with the right solution for this please?

**Comment by willpower232** [comment]
Have a look at #1227 #255 and maybe #1298

**Reply by XenyFord** [reply]
Mr. Will,

Sorry to say the whole posts(#1227 #255 and maybe #1298) you have suggested above are not really working...Instead of good results it made something gone wrong...

And finally..
I'm not understanding whats wrong with me at postal. Because, No bounced emails are forwarded to me from postal, even though I added(no-reply@mydomain.com) to Routes as well as ENDPOINT, and tested several times sending emails to "SoftBounce@bounce-testing.postmarkapp.com", but all the bounce messages are still showing as failed, but nothing forwarded to me on no-reply@mydomain.com.

Below are the screens for your reference.. 
Routes & ENDPOINT: [https://prnt.sc/1qi6elc](url)
BouncedEmails[https://prnt.sc/1qi6uoy](url)

Kindly help me with it If possible little detailed, step by step on how to add email and etc to get the bounced emails forwarded to me as quickly as it hits to the postal...

Thank you in advance!

**Reply by XenyFord** [reply]
Hi Mr. will,

Another thing, If you can help me..

When I use two different commands in crontab as mentioned below to be executed on server reboot, postal is not properly started, and one of the postal elements(postal_worker_1) was stuck, not started.

@reboot iptables-restore < /xyz/iptables/xyg.conf
@reboot postal start

So, Can you please help me with a perfect way(command) to make these two lines executed properly without any human intervention on server reboot..

A heartfelt thanks in advance!

**Comment by XenyFord** [comment]
Mr. Will,

Sorry to say the whole posts(#1227 #255 and maybe #1298) you have suggested above are not really working...Instead of good results it made something gone wrong...

And finally..
I'm not understanding whats wrong with me at postal. Because, No bounced emails are forwarded to me from postal, even though I added(no-reply@mydomain.com) to Routes as well as ENDPOINT, and tested several times sending emails to "SoftBounce@bounce-testing.postmarkapp.com", but all the bounce messages are still showing as failed, but nothing forwarded to me on no-reply@mydomain.com.

Below are the screens for your reference..
Routes & ENDPOINT: https://prnt.sc/1qi6elc
BouncedEmails https://prnt.sc/1qi6uoy

Kindly help me with it If possible little detailed, step by step on how to add email and etc to get the bounced emails forwarded to me as quickly as it hits to the postal...

Thank you in advance!
